server/src/test/java/org/elasticsearch/cluster/serialization/DiffableTests.java

```diff
@@ -87,7 +86,7 @@ protected MapDiff<Integer, String, Map<Integer, String>> readDiff(StreamInput in
     }
 
     public void testImmutableOpenMapDiff() throws IOException {
-        new ImmutableOpenMapDriver<TestDiffable>() {
+        new ImmutableMapDriver<TestDiffable>() {
```

Comment:
We probably shouldn't be changing this test, it explicitly tests if we can correctly make diffs on ImmutableMaps. There's still various metadata code that uses ImmutableMap and therefore ImmutableMap diffs.